README.md

```diff
@@ -1 +1,35 @@
-Orbit
+# Orbit - Automating AltCampus application process
+
+## About Orbit
+This application aims to smoothen the applicant  application process at AltCampus. Users can login to see their application progress in real time. Users can download and submit assignments, solve questions and schedule an interview. Admins can grade the submited assignments and questions. Based on the score(both the objectively calculated and the one assigned by the admins) it will be determined whether the user is qualified for the interview or not.
```

Comment:
"smoothen the applicant  application process" doesn't seem right. 

Break the paragraph into two paras. After "Admins can ...." should be another paragraph.

Please fix it.

Reply:
@amanregu @singhabhinav96 please check the review and make fixes.

Reply:
@prank7 Updated as per the requirement. Please review.